### ISSUE TITLE
Write tests to make sure the scopes properly identify various values

### ISSUE THREAD

**Issue by connorshea** [connorshea/mdn-compat-data-explorer#25]
From #12: 

>The graph values don't quite add up when there are more complex browser support objects, e.g. it handles `{ version_added: true }` but I'm not 100% sure it handles `{ version_added: true, prefix: '-webkit'}` and it definitely doesn't handle support objects that are arrays of these hashes.

I'm pretty sure it handles the second case properly now (not the third though), but it'd be useful to make sure it _stays_ true as development of the Rails app continues.

Rather than using data from `data-test.json`, which could cause failures any time the data is modified, I think it'd be a good idea to create the Feature objects when running the test suite using something like [factory_bot](https://github.com/thoughtbot/factory_bot).

I was initially hesitant to do this because I wouldn't be able to make sure the JSON that gets passed by `seeds.rb` matches the JSON used in the tests, but I think that the benefits outweigh the potential problems. Hopefully I can come up with some way to make sure the data actually _in_ the database and the data in the tests match up.

tl;dr use fixtures/factories to generate Features and write simpler, more resilient tests.
